common/sgx/sgxcertextensions.c

```diff
@@ -203,11 +203,8 @@ static oe_result_t _read_extension(
 
 static void _trace_hex_dump(const char* tag, const uint8_t* data, size_t size)
 {
-    if (oe_get_current_logging_level() >= OE_LOG_LEVEL_INFO)
-    {
-        OE_TRACE_INFO("%s = ", tag);
-        oe_hex_dump(data, size);
-    }
+    OE_TRACE_INFO("%s = ", tag);
+    oe_hex_dump_trace(OE_LOG_LEVEL_INFO, data, size);
```

Comment:
I think we should retain the behavior where log dumping depends on level instead of defaulting to info

Reply:
The level argument is passed here to let the logging capabilities know that we got to print this value only when current logging level > passed in logging level. In this case, OE_LOG_LEVEL_INFO